src/query/frontend/semantic/required_privileges.cpp

```diff
@@ -29,9 +29,9 @@ class PrivilegeExtractor : public QueryVisitor<void>, public HierarchicalTreeVis
 
   void Visit(AuthQuery &) override { AddPrivilege(AuthQuery::Privilege::AUTH); }
 
-  void Visit(ExplainQuery &query) override { query.cypher_query_->Accept(*this); }
+  void Visit(ExplainQuery &query) override { query.cypher_query_->Accept(dynamic_cast<QueryVisitor &>(*this)); }
```

Comment:
Just a question, is dynamic cast necessary? I read recently it can be really slow

Reply:
it's done only once though, I think it shouldn't be degrading performance. For some reason static cast does not work though